db/migrate/20220324143540_add_validation_message_to_dialog_fields.rb

```diff
@@ -0,0 +1,5 @@
+class AddValidationMessageToDialogFields < ActiveRecord::Migration[6.0]
+  def change
+    add_column :dialog_fields, :validator_message, :string
```

Comment:
How about `failed_validation_message`?  That or something that matches the terminology we have in the UI for the message field for the failed validation.  If there's in intention to use this field for general informational purposes, the current value here is ok but if it's solely for failure messages, included failure or failed in the field would be helpful.

Reply:
`failed` was already removed from the UI since the message only helps the user input the data.

Reply:
@jrafanie you all set with this answer?

Reply:
another alternative is `validate_failure_message`